packages/react-avatar/src/AvatarSize.stories.tsx

```diff
@@ -3,19 +3,19 @@ import { Avatar, AvatarProps } from './index';
 
 export const Size = (props: Partial<AvatarProps>) => (
```

Comment:
Should we either remove the `props` from the arguments of the function here or re-add `{...props}` in the components below?

Reply:
The stories need a more thorough pass to fix up, but it's out of scope of this PR. I'm planning to do that in a follow-up PR.